boot-qemu.py

```diff
@@ -669,7 +673,7 @@ def launch_qemu(cfg):
                 gdb_cmd = [gdb_bin]
                 gdb_cmd += [kernel_location.joinpath("vmlinux")]
                 gdb_cmd += ["-ex", "target remote :1234"]
-                subprocess.run(gdb_cmd)
+                subprocess.run(gdb_cmd, check=False)
```

Comment:
Do we want `check=True`? What happens if we fail to launch gdb?

Reply:
I don't think we want `check=True` because that will kill the script outright. If we just ignore the `gdb` failure with `check=False` instead, the user will just be prompted to retry with the error right above it, so it is up to them what to do. I do not care much though, I can change that to `check=True`.

Reply:
Merging for now, as this is currently how things are. I can adjust it in the future if desired.